src/components/MenuItem/MenuItem.jsx

```diff
@@ -46,19 +58,33 @@ export class MenuItem extends React.Component {
 
   action() {
     const { addToOrder, name } = this.props;
-    const { total, bread, isHot } = this.state;
+    const { total, bread, isHot, owner } = this.state;
 
     addToOrder(
       name,
       total,
       bread,
-      isHot
+      isHot,
+      owner
     );
   }
 
+  handleClick() {
+    this.setState({ modalOpen: true });
+  }
+
+  closeModal() {
+    this.setState({ modalOpen: false });
+  }
+
+  addOrderOwner(owner) {
```

Comment:
```suggestion
  updateOrderOwner(owner) {
```

It can edit as well as just add 👍